manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "Video streaming platform using P2P directly in the web browser, connected to a federated network",
         "fr": "Plateforme de diffusion vidéo par P2P directement dans le navigateur, et connectée à un réseau fédéralisé"
     },
-    "version": "3.1.0~ynh6",
+    "version": "3.2.1~ynh6",
```

Comment:
Shouldn't the version be `3.2.1~ynh1`?
(comment also applicable to versions above)

Reply:
yes would be better to be `3.2.1~ynh1`